lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/declaration/kopropertydeclaration/KoPropertyDeclarationForReceiverTypeTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.lemonappdev.konsist.core.declaration.kopropertydeclaration
+
+import com.lemonappdev.konsist.TestSnippetProvider
+import org.amshove.kluent.assertSoftly
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
+class KoPropertyDeclarationForReceiverTypeTest {
+    @Test
+    fun `property-without-receiver-type`() {
+        // given
+        val sut = getSnippetFile("property-without-receiver-type")
+            .properties(includeNested = true)
+            .first()
+
+        // then
+        assertSoftly(sut) {
+            receiverType shouldBeEqualTo null
+            hasReceiverType() shouldBeEqualTo false
+            hasReceiverType("Int") shouldBeEqualTo false
+        }
+    }
+
+    @Test
+    fun `property-with-simple-receiver-type`() {
```

Comment:
From parsing perspective there is no difference if the type is simple type or complex tyoe - these test should be enough:
- property-with-receiver-type
- property-without-receiver-type

Also update `KoFunctionDeclarationForReceiverTypeTest`